### DISCUSSION TITLE
blog/how-to-write-a-good-revision-letter

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#63] [Announcements]
# blog/how-to-write-a-good-revision-letter

Dealing with a major revision is a challenging process in the life of doctoral students. In this post, I share my personal system and some useful resources that may help you to navigate the academic revision process with panache.

https://www.cesarsotovalero.net/blog/how-to-write-a-good-revision-letter.html

**Comment by janenguyen131** [comment]
Hello! Just want to say thank you very much. Your post has been a life-savior. Last year I was submitting my first first-author paper to an isi journal and received major revision. I was a noobie undergrad and could not ask anyone, even my supervisor since he was having some personal issues. Luckily I stumbled upon your sharings. I replied to every reviewer's comments following your style, got through major revision without any objections, and only had to fix formatting issues in minor revision stage before the paper was finally accepted this month! Thanks a lot and I wish you all the best in your future endevours!

**Reply by cesarsotovalero** [reply]
Hi @janenguyen131, Thank you for you comment. I'm glad that you find this post useful. There is a lot of ~~hidden~~ knowldege in academic research that's swept under the rug and unspoken, and revision letters are the typical example as they ~~almost~~ never go public. I hope more people will benefit from this post as you did. Wish you the best in your journey!
